.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+#### Ticket
+
+[TICKET_NUMBER](https://vimean.atlassian.net/browse/TICKET_NUMBER)
+
+- *Note: this is required for Vimeo staff only. If not applicable to your PR, use "N/A".*
+
+#### Pull Request Checklist
+
+- [ ] Resolved any merge conflicts
+- [ ] No build errors or warnings are introduced
```

Comment:
@fyell the thought was folks shouldn't expect us to review if a build is broken or has warnings